Add explorer

reisner opened the pull request:
Added Term explorer to Analyze flow. Bonus: If you select a term with >= 10 occurences, it shows a density plot of sentiment. 

![term](https://user-images.githubusercontent.com/490216/36569242-3d7608c6-17ea-11e8-90c4-59bdeb7c2f06.png)

ben-gready commented:
Few points:
- I personally like density plots, but I think that our users will be much more familiar with histrograms, so my vote would be to use a histogram.
- could argue that the summary stats and plot should belong in the lower pain
- could consider ordering the words by frequency... not sure if that's a good idea though

reisner commented:
@ben-gready I moved stuff to the bottom pane, and I added a histogram (why not both!)

As for ordering by frequency, I didn't want to do that since you can do that with the barplots. 

![termstats](https://user-images.githubusercontent.com/490216/36605105-f0331070-187c-11e8-8a2c-33f9a830a393.png)

ben-gready commented:
looks good!
